src/components/atoms/SelectInput/Option/Option.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import typography from 'styles/typography';
+import colors from 'styles/colors';
+import { CheckedOptionIcon, UnCheckedOptionIcon } from 'assets/index';
```

Comment:
Please remove this `index`

```suggestion
import { CheckedOptionIcon, UnCheckedOptionIcon } from 'assets';
```